nx-X11/programs/Xserver/mi/mibank.c

```diff
@@ -175,8 +175,8 @@ typedef struct _miBankQueue
         (*pScreenPriv->BankInfo.SetDestinationBank)(pScreen, (_no)) - \
         (pScreenPriv->BankInfo.BankSize * (_no)))
 
-#define ALLOCATE_LOCAL_ARRAY(atype, ntype) \
-    (atype *)ALLOCATE_LOCAL((ntype) * sizeof(atype))
+#define malloc_ARRAY(atype, ntype) \
+    (atype *)malloc((ntype) * sizeof(atype))
```

Comment:
Here search+replace failed, I first thought. How does upstream address this code change? Your approach feels like an artifact.... I looked at X.org xserver code and they actually have xalloc_ARRAY everywhere, but later on drop mibank.c completely. I am aware of an issue report for that. So let's leave malloc_ARRAY as is for now, although it really feels like a search+replace artifact.